content/webapp/pages/article.js

```diff
@@ -21,60 +23,96 @@ type Props = {|
-    url: `/series/${series.id}`,
-    text: series.title || '',
-    prefix: 'Part of'
-  })));
+  const breadcrumbs = {
+    items: [{
+      url: '/stories',
+      text: 'Stories'
+    }].concat(article.series.map(series => ({
+      url: `/series/${series.id}`,
+      text: series.title || '',
+      prefix: 'Part of'
+    })))
+  };
 
-  const Header = (
-    <TextLayout>
-      <HeaderText
-        Breadcrumb={<Breadcrumb items={crumbs} />}
-        Heading={<h1 className='h1 inline-block no-margin'>{article.title}</h1>}
-        DateInfo={null}
-        InfoBar={
-          <div className={classNames({
-            'flex': true,
-            [font({s: 'HNM5'})]: true
-          })}>
-            <p className={classNames({
-              [spacing({s: 1}, {margin: ['right']})]: true,
-              [spacing({s: 0}, {margin: ['bottom']})]: true
-            })}>
-              <b>By </b>
-              {article.contributors.map(({ contributor }, i, arr) => (
-                <Fragment key={contributor.id}>
-                  <a
-                    className={'plain-link font-green'}
-                    href={`/${contributor.type}/${contributor.id}`}>
-                    {contributor.name}
-                  </a>
+  // TODO: do this properly
+  const labels = {
+    labels: [{
+      url: null,
+      text: 'Essay'
+    }]
+  };
 
-                  {i < arr.length - 1 && ', '}
-                </Fragment>
-              ))}
-            </p>
-            <div className={classNames({
-              'font-pewter': true,
-              [font({s: 'HNL5'})]: true
-            })}>
-              <HTMLDate date={new Date(article.datePublished)} />
-            </div>
-          </div>
-        }
-        Description={article.summary ? <PrismicHtmlBlock html={article.summary} /> : null}
-      />
-    </TextLayout>
-  );
+  const genericFields = {
+    id: article.id,
+    title: article.title,
+    contributors: article.contributors,
+    contributorsTitle: article.contributorsTitle,
+    promo: article.promo,
+    body: article.body,
+    promoImage: article.promoImage,
+    promoText: article.promoText,
+    image: article.image,
+    squareImage: article.squareImage,
+    widescreenImage: article.widescreenImage
+  };
+  const standfirst = article.body.find(slice => slice.type === 'standfirst');
+  const ContentTypeInfoComponent = standfirst &&
+    <Fragment>
+      <div className={classNames({
+        'first-para-no-margin': true,
+        [spacing({s: 1}, {margin: ['top']})]: true
+      })}>
+        <PrismicHtmlBlock html={standfirst.value} />
+      </div>
+      <div className={classNames({
+        'flex': true,
+        'flex--h-baseline': true,
+        [font({s: 'HNM5'})]: true
+      })}>
+        <p className={classNames({
+          [spacing({s: 1}, {margin: ['top']})]: true,
+          [spacing({s: 1}, {margin: ['right']})]: true,
+          [spacing({s: 0}, {margin: ['bottom']})]: true
+        })}>
+          <b>By </b>
```

Comment:
I think this will just bolded by the browser – would probably be better to use our font classes